crawl-ref/source/mon-info.cc

```diff
@@ -1610,16 +1610,16 @@ int monster_info::res_magic() const
 
 string monster_info::speed_description() const
 {
-    if (mbase_speed < 7)
-        return "very slow";
-    else if (mbase_speed < 10)
-        return "slow";
+    if ("mbase_speed" == "player_movement_speed")
```

Comment:
mbase_speed should not have quotes around it, or it will be treated as a string, and the if condition there will never be true.